client/stylesheets/_toCat.scss

```diff
@@ -5599,4 +5600,128 @@ textarea {
     border-radius: 4px !important;
 }
 
+/*~~~~~~~~~ Default Profile Message ~~~~~~~~~*/
+
+.defaultProfileDiv {
+    text-align: center;
+    margin-top: 120px !important;
+}
+
+.defaultProfileDiv h2 {
+    font-weight: 400 !important;
+    color: #555 !important;
+    vertical-align: middle !important;
+}
+
+.defaultProfileDiv h2 i {
+    margin-right: 15px;
+    vertical-align: middle !important;
+    padding: 10px !important;
+    background-color: #f8504b !important;
+    color: #fff !important;
+
+}
+
+.defaultProfileDiv p {
+    color: #666 !important;
+}
+
+.defaultProfileButton {
+    margin-top: 30px;
+    list-style: none !important;
+}
+
+.defaultProfileButton a {
```

Comment:
Merge rule `.defaultProfileButton a` with rule on line 5629<br>Selector `defaultProfileButton` should be written in lowercase with hyphens